internal/pkg/text/template/parse/testdata/nested_conditional_with_if_definition/templates/NestedConditionalWithIfDefinition.yml

```diff
@@ -13,4 +13,7 @@ condition1IsFalse
   {{ range .Values.someList }}
   {{ . }}
   {{ end }}
+  {{- range $key, $value := .Values.metrics.serviceMonitor.selector }}
```

Comment:
Should we add a nested range for testing?

Reply:
Already I  have added with nested_conditional_with_if

Reply:
I meant a nested loop.  I.e., a loop inside a loop.